Makefile

```diff
@@ -279,7 +279,7 @@ importcheck:
 static-check: gocyclo govet importcheck
 	# check for unused code using staticcheck binary
```

Comment:
This comment is out of date, it should be updated to explain what "-ST*" and "-SA1019" do and why we are excluding them.

Reply:
Made a new commit with update comment.